### ISSUE TITLE
Gyroscope support?

### ISSUE THREAD

**Issue by chrislachance** [GaryOderNichts/Bloopair#42]
I think you probably are aware, but gyroscope support would be great in the future if not already supported.

**Comment by dude98** [comment]
The Wii U pro controller doesn't have a Gyroscope, so unless they eventually start emulating a gamepad or wiimote plus (or wii motion plus), there's no point in adding a gyroscope.

**Comment by monkeyman4412** [comment]
There is a use for this, albeit, one tiny microscopic case. Splatoon's local play. It would be hell of a lot less janky, than what the game asks you to do. (which is to essentially ruberband a wiimotion plus controller to the bottom of your full controller, to play locally)

**Comment by monkeyman4412** [comment]
There is, unironically, a use for this. Setup the gyro as a wiimotion plus controller ontop of the pro controller. And now you can use that configuration for splatoon's local multiplayer

**Comment by chrislachance** [comment]
This is the exact reason I'm hoping for it. My kids love that mode.

**Comment by Sentineluno** [comment]
Me too, was just playing breath of the wild but it's not comfortable to play without gyro aim for the bow aiming and even certain shrines, it may not be possible but if there is i would love to see this being a thing.

**Comment by anegrin** [comment]
>  And i mean when using for example a switch pro controller.

yeah this would be great as it would mean you can also use a 8bitdo pro 2 and gyro... +1 for the feature

**Comment by anegrin** [comment]
oh here's why this can't happen

https://github.com/GaryOderNichts/Bloopair/issues/33#issuecomment-966099002

**Comment by chrislachance** [comment]
That makes sense, but I believe (a least for Splatoon) it takes the input of TWO controllers; gyro of one, inputs of the other.
Wonder if that's possible.

(How it worked - https://www.youtube.com/watch?v=vBlNDDZRf-4)
